MAINTAINERS

```diff
@@ -0,0 +1,16 @@
+Avocado-vt maintainers
+======================
+
+The intention of this file is not to establish who owns what portions of the
+code base, but to provide a set of names that developers can consult when they
+have a question about a particular subset and also to provide a set of names
+you can look for on github for any pull requests you might want to get approved.
+
+In general, if you have a question, you should send an email to the Virt Test
+development mailing list and not any specific individual privately.
+
+L: Avocado-devel <avocado-devel@redhat.com>
+M: Yanbing Du <ydu@redhat.com>
+M: Jiangang Wei <weijg.fnst@cn.fujitsu.com>
+M: Guannan Wayne Sun <gsun@redhat.com>
+M: Hao Liu <hliu@redhat.com>
```

Comment:
@ldoktor I think xutian and clebergnu are missing in this.

And currently I don't have the merge permission for avocado-vt, it's lost from virt-test to avocado-vt migration. I'm still actively involved in the project, recently on stabilizing tp-libvirt testing which reply on this.